qiskit/transpiler/passes/optimization/commutative_inverse_cancellation.py

```diff
@@ -23,19 +23,20 @@
 class CommutativeInverseCancellation(TransformationPass):
     """Cancel pairs of inverse gates exploiting commutation relations."""
 
-    def __init__(self, max_qubits: int = 4, upto_phase_optimization: bool = False):
+    def __init__(self, matrix_based: bool = False, max_qubits: int = 4):
         """Initialize CommutativeInverseCancellation pass.
 
```

Comment:
Continuing the nit-pick: it might be nice to remove this, as we do with lots of classes, if the initializer doesn't say anything special.
```suggestion
        """
```

Reply:
Removed.